homeassistant/components/template/template_entity.py

```diff
@@ -473,6 +473,10 @@ def _handle_results(
                     event, update.template, update.last_result, update.result
                 )
 
+        self._write_state_and_update_preview()
```

Comment:
The preview should not incorporate delay on or delay off.  In many cases, people use large delays, this would force people to wait those large delays in order to see the preview.

Reply:
Makes sense. So our preview should simply display the value after the delay on / delay off (currently it shows "unknown" and then "on/off" after the delay)?

Will make some changes!

Reply:
Done, please take another look, thanks!